apps/site/pages/en/blog/release/january-2025-security-releases.md

```diff
@@ -0,0 +1,38 @@
+---
+date: 2025-01-14T03:00:00.000Z
+category: vulnerability
+title: Tuesday, January 21, 2025 Security Releases
+slug: january-2025-security-releases
+layout: blog-post
+author: The Node.js Project
+---
+
+# Summary
+
+The Node.js project will release new versions of the 23.x, 22.x, 20.x, 18.x
+releases lines on or shortly after, Tuesday, January 21, 2025 in order to address:
+
+- 1 high severity issues.
+- 2 medium severity issues.
+
+## Impact
+
+The 20.x release line of Node.js is vulnerable to 1 high severity issues, 2 medium severity issues.
+The 22.x release line of Node.js is vulnerable to 1 high severity issues, 2 medium severity issues.
+The 23.x release line of Node.js is vulnerable to 1 high severity issues, 2 medium severity issues.
+The 18.x release line of Node.js is vulnerable to 2 medium severity issues.
```

Comment:
Non-blocking nit: It would be nice if this was sorted (probably with 23.x first descending down to 18.x to match the order in the first sentence of the Summary).

Reply:
Hmm, It was automated, I'll create an issue and fix it.